trunk/workflow-manager/src/main/java/org/mitre/mpf/wfm/camel/operations/detection/trackmerging/TrackMergingPlan.java

```diff
@@ -27,17 +27,35 @@
 package org.mitre.mpf.wfm.camel.operations.detection.trackmerging;
 
 public class TrackMergingPlan {
-	/** Indicates whether track merging is to be performed. */
+	/**
+	 * Indicates whether to merge tracks. If track merging is turned off, minGapBetweenTracks is invalid,
+	 * but minTrackLength is still respected. */
 	private final boolean mergeTracks;
+	/** Indicates the shortest track length to keep. */
+	private final int minTrackLength;
+	/** The allowable distance between similar tracks without merging. */
+	private final int minGapBetweenTracks;
+
 	public boolean isMergeTracks() { return mergeTracks; }
 
+	public int getMinTrackLength() {
+		return minTrackLength;
+	}
+
+	public int getMinGapBetweenTracks() {
+		return minGapBetweenTracks;
+	}
+
 	/** For images and videos, the number of frames at which the input medium was sampled. For audio, the number seconds between samples. */
 	private final int samplingInterval;
 	public int getSamplingInterval() { return samplingInterval; }
```

Comment:
This issue is part of the concern I've raised regarding scope.  This was a pre-existing format issue that was not related to the change for this pull request.  There are a few other requests that are similar in here. In order to make the deadline of tomorrow, can we push this off into a separate task that that will address formatting outside the scope of my code changes?